### PR TITLE
Created list contacts UI

### LINKED ISSUE #14: creating list contacts ui

### PR DESCRIPTION
**Pull request by nancy-binani**

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/Table/index.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+import { Table as NeetoUITable, Pagination } from "neetoui";
+
+import { buildContactsColumnData } from "./utils";
+
+import { CONTACT_DETAILS } from "../constants";
+
+const Table = () => (
+  <>
+    <NeetoUITable
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Table/index.jsx]: Why do we need this fragment here?

**Review by yedhink — CHANGES_REQUESTED** [review]
@nancy-binani After the following fix is made, go ahead and merge this PR.

File: app/javascript/src/components/Dashboard/Contacts/Table/index.jsx (hunk 1 above)
Comment (nancy-binani) [reply app/javascript/src/components/Dashboard/Contacts/Table/index.jsx]: There is a pagination component as well, that's why I added the React Fragment.

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Contacts/Table/index.jsx]: Got it. Didn't see the Pagination component.

**Pull request merged**